scrollphat/auth.py

```diff
@@ -0,0 +1,4 @@
+consumer_key = 'WwXsQ5djrUXBE1I6QKCt3MvDQ'
```

Comment:
Does this need to be on Github

Reply:
check again - have definitely deleted this on the branch now